mlflow/models/utils.py

```diff
@@ -122,9 +141,13 @@ def _save_example(mlflow_model: Model, input_example: ModelInputExample, path: s
 
     The metadata is a dictionary with the following fields:
       - 'artifact_path': example path relative to the model directory.
-      - 'type': Type of example. Currently the only supported value is 'dataframe'
-      - 'pandas_orient': Determines the json encoding for dataframe examples in terms of pandas
-                         orient convention. Defaults to 'split'.
+      - 'type': Type of example. Currently the supported values are 'dataframe' and 'ndarray'
+      -  This field depends on the type of data being stored
```

Comment:
Nit: Should we say `one of` instead of `This field depends ...`? One of is used in protobuf.